src/lib/core/core.gni

```diff
@@ -99,6 +99,8 @@ declare_args() {
   chip_tlv_validate_char_string_on_write = true
   chip_tlv_validate_char_string_on_read = false
 
+  chip_max_paths_per_invoke = 1
```

Comment:
Could this cause issues with apps that are built with build systems where they link against libCHIP and the app can have a different config in a different build system? There may be a mismatch between them.